cloud-sql-proxy/test_cloud_sql.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import random
+
+from parameterized import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class CloudSqlProxyTestCase(DataprocTestCase):
+    COMPONENT = 'cloud-sql-proxy'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/cloud-sql-proxy/cloud-sql-proxy.sh'
+    TEST_SCRIPT_LOCATION = 'dataproc-initialization-actions/cloud-sql-proxy'
+    TEST_SCRIPT_FILE_NAME = 'pyspark_metastore_test.py'
+    SCOPES = 'sql-admin'
+    DB_NAME = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        _, region, _ = cls.run_command("gcloud config get-value compute/region")
+        cls.REGION = region.strip() or "global"
+        _, project, _ = cls.run_command("gcloud config get-value project")
+        project = project.strip()
+        cls.PROJECT_METADATA = '{}:{}'.format(project, cls.REGION)
+
+    def setUp(self):
+        super().setUp()
+        self.DB_NAME = "test-{}-db".format(random.randint(1, 10000))
```

Comment:
Let's use timestamps instead of random numbers for naming the entities (db).

Reply:
Ok thx, I will apply those suggestions.